contrib-compiler.md

```diff
@@ -0,0 +1,99 @@
+
+
+Thanks for interest in contributions.
+
+Note currently our philosphy is to make release build as easy as possible, while dev build is fairly complex at this time.
+
+## Set up dev dependencies
+
+- Having [https://opam.ocaml.org/](opam) installed
+
+    opam switch 4.02.3+buckle-master # use our OCaml compiler
```

Comment:
GitHub's Markdown parser formats these two lines as a single line.

Also, is <1> meant to be a reference to something?